src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
+        public WebhookSeries Series { get; internal set; }
+        public WebhookEpisodeFile EpisodeFile { get; internal set; }
+        public WebhookEpisode Episode { get; internal set; }
```

Comment:
can we make this a list, that way in the future when we add support for bulk notifications (you get notification for a season pack) we don't have to break the API.